aimmdb/ingest/aimmdb_backup.py

```diff
@@ -0,0 +1,102 @@
+import json
+import os
+import pandas as pd
+
+def download_aimmdd_data(c):
+    """
+    Navigate through a tiled client and download all the entries that are found
+
+    Parameters
+    ----------
+    c : tiled.client
+
+    Returns
+    -------
+    None.
+
+    """
+    
+    if not os.path.exists("files/"):
+        os.mkdir("files/")
+    
+    for key in c['dataset'].keys():
```

Comment:
This is perfectly fine. It's useful to know that iterating through a dict-like object iterates through its keys, so you can just write:

```suggestion
    for key in c['dataset']:
```